community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -398,6 +398,9 @@ private static String defaultPageCacheMemory()
             + "_NOTE: This feature is only available in the Neo4j Enterprise Edition_." )
     public static final Setting<Boolean> log_queries = setting("dbms.querylog.enabled", BOOLEAN, FALSE );
 
+    @Description("Path of the run directory")
+    public static final Setting<File> run_directory = setting("dbms.directories.run", PATH, "run");
```

Comment:
This should go in `ServerSettings` as it's server only -- not interesting to the kernel.